Exported Deck Pile Labels

zombieCraig opened the pull request:
Exports a Deck Label so you can easily label your deck piles from your Board scene.

db0 commented:
But, you can do that anyway, by changing its label text :) You just want something more obvious?

zombieCraig commented:
Personally I didn't find it obvious on where to control the label.  Since I had to create several piles my workflow as to:
+ Open my Board
+ New Instanced Scene (Picked Pile)
+ Click on your new deck pile and in the Inspector, Changed the settings for location, shuffle style and other behaviors but I couldn't set the label of the deck that the user would see.
That was the only major piece I saw missing from the export var list.  You would name your Desk on instance so that doesn't get changed.  I did consider automatically using the name of the Node by default.   But I changed by mind as the code looked a bit less obvious than your default of just calling it "Deck"

You are correct that this PR is not a bug fix but a quality of life improvement.

db0 commented:
I decided to merge this, for potential ease of use. Let's see if it's useful or confusing.
